modules/ui/src/main/resources/lucuma-css/lucuma-ui.scss

```diff
@@ -5,13 +5,13 @@
   --button-padding: 0.78571429em;
   --compact-button-padding: 0.39285714em;
 
-  --button-mini-font-size: 0.78571429rem;
-  --button-tiny-font-size: 0.85714286rem;
-  --button-small-font-size: 0.92857143rem;
-  --button-large-font-size: 1.14285714rem;
-  --button-big-font-size: 1.28571429rem;
-  --button-huge-font-size: 1.42857143rem;
-  --button-massive-font-size: 1.71428571rem;
+  --pl-mini-font-size: 0.78571429rem;
```

Comment:
I guess we need to propagate these changes to explore?

Reply:
Yes, although I don't think those were being used directly. Well, until my next PR in Explore, that is.